src/Azure.Functions.Cli/Actions/LocalActions/CreateFunctionAction.cs

```diff
@@ -430,7 +435,7 @@ private bool IsNewNodeJsProgrammingModel(WorkerRuntime workerRuntime)
                         var packageJsonData = FileSystemHelpers.ReadAllTextFromFile(Constants.PackageJsonFileName);
                         var packageJson = JsonConvert.DeserializeObject<JToken>(packageJsonData);
                         var funcPackageVersion = packageJson["dependencies"]["@azure/functions"];
-                        if (new Regex("^[^0-9]*4").IsMatch(funcPackageVersion.ToString()))
+                        if (funcPackageVersion != null && new Regex("^[^0-9]*4").IsMatch(funcPackageVersion.ToString()))
```

Comment:
It was throwing exception in v3. I know that we assume `false` when there is an exception. Trying to avoid an exception in the first place.